notebooks/DrizzlePac/align_to_catalogs/align_to_catalogs.ipynb

```diff
@@ -4,42 +4,80 @@
    "cell_type": "markdown",
```

Comment:
<p>Descriptive variable naming makes the code self-documenting and easier to read. Similar to above code cell,  adding comments explaining the purpose of each block would significantly enhance the readability.</p>

<pre class="ql-syntax" spellcheck="false">EXT_0_KEYWORDS = ['DETECTOR', 'EXPTIME'] # extension 0 keywords.
EXT_1_KEYWORDS = ['WCSNAME', 'NMATCHES', 'RMS_RA', 'RMS_DEC'] # extension 1 keywords.

# Define the detector plate scales in arcsec per pixel.
DETECTOR_SCALES = {
&nbsp;&nbsp;'IR': 0.1283,&nbsp;
&nbsp;&nbsp;'UVIS': 0.0396,&nbsp;
&nbsp;&nbsp;'WFC': 0.05
}

formatted_data = {}
column_data = defaultdict(list)

for fits_file in sorted(glob.glob('*flc.fits')):
&nbsp;&nbsp;column_data['filename'].append(fits_file)
&nbsp;&nbsp;header0 = fits.getheader(fits_file, 0)
&nbsp;&nbsp;header1 = fits.getheader(fits_file, 1)
&nbsp;&nbsp;&nbsp;
&nbsp;&nbsp;for keyword in EXT_0_KEYWORDS:
&nbsp;&nbsp;&nbsp;&nbsp;column_data[keyword].append(header0[keyword])
&nbsp;&nbsp;for keyword in EXT_1_KEYWORDS:&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
&nbsp;&nbsp;&nbsp;&nbsp;if 'RMS' in keyword:
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;value = np.around(header1[keyword], decimals=1)
&nbsp;&nbsp;&nbsp;&nbsp;else:&nbsp;
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;value = header1[keyword]
&nbsp;&nbsp;&nbsp;&nbsp;column_data[keyword].append(value)
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
&nbsp;&nbsp;for rms_keyword in ['RMS_RA', 'RMS_DEC']:
&nbsp;&nbsp;&nbsp;&nbsp;rms_value = header1[rms_keyword] / 1000 / DETECTOR_SCALES[header0['DETECTOR']]
&nbsp;&nbsp;&nbsp;&nbsp;column_data[f'{rms_keyword}_pix'].append(np.round(rms_value, decimals=2))

wcstable = Table(column_data)
wcstable
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/hst_notebooks/pull/265/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
Agreed, we have clarified the explanation for this cell and provided a link to Section 2.4 of the WFC3 Data Handbook, which provides a full list and explanation of the available header keywords. All other cells contain brief markdown explanations.

Reply:
Agreed, I have updated the cell block to the suggestion provided above, with minor changes to syntax.